### PR TITLE
Ia-1611: completeness table front end

### PR DESCRIPTION
**Pull request by niconoe**
First draft of a new completeness table

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests

## Changes

Base implementation (front + back)

## How to test

Test that there is a new "completeness" entry in the menu, and that some values are displayed without error (first iteration, we'll then await feedback from Moritz!)

## Notes

As discussed in the squad, it would be great if that was merged very soon so Moritz can review from the user's perspective and it still has a chance t be included in tomorrow's release.

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetCompletnessStats.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+export type CompletenessGETParams = UrlParams & {
+    parentId?: string;
+    formId?: string;
+    orgUnitTypeId?: string;
+};
+
+const apiParamsKeys = ['order', 'page', 'limit', 'search'];
+
+const getCompletenessStats = async (params: CompletenessGETParams) => {
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetCompletnessStats.ts]: return is not typed

**Review by beygorghor — APPROVED** [review]
Nothing blocking:
- commented code
- Maybe make the difference between the completeness entries in the menu
- Maybe a better url can be `forms/completeness/stats` instead

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetCompletnessStats.ts (hunk 1 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetCompletnessStats.ts]: nothing gets past

Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/completenessStats/hooks/api/useGetCompletnessStats.ts]: The worst part is that the type already exists

**Pull request merged**